nntrainer/layers/cl_layers/fc_layer_cl.h

```diff
@@ -101,12 +101,19 @@ class FullyConnectedLayerCl : public LayerImpl {
    */
   void setProperty(const std::vector<std::string> &values) override;
 
+  static bool registerClKernels() { return true; };
```

Comment:
Is it NYI or always true?

Reply:
There's no need to register kernel for `fc_layer_cl`, since it works with blas kernel only (cl blas kernels are registered separately). Thus, it is always true.